What's-new-in-TypeScript.md

```diff
@@ -37,9 +37,32 @@ tsc foo.ts --outFile /dev/stdout | pretty-js
 
 Specifying `--outFile` in conjunction with `--module amd` or `--module system` will concatenate all modules in the compilation into a single output file containing multiple module closures.
 
-## New compiler flag : `--allowSyntheticDefaultExports`
+## The new `--allowSyntheticDefaultImports` compiler flag
 
-When `--allowSyntheticDefaultImports` is specified, it indicates that the module loader performs some kind of synthetic default import member creation not indicated in the imported .ts or .d.ts. We infer that the default member is either the export= member of the imported module or the entire module itself should that not be present. System modules have this flag on by default.
+Default exports are an ES2015 feature for when modules are meant to have a primary usage.
```

Comment:
Perhaps rather:

> Default exports are an ES1015 feature which allow users to mark an export as the containing module's "default" export so that it can be imported with a shorthand syntax.